### DISCUSSION TITLE
<title>I can’t log in

### DISCUSSION THREAD

**Discussion by ikokkari** [community/community#41459] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I cleared my browser cache recently on my Mac desktop.

I know my GitHub password but now can’t log in because after username and password, it asks me to enter a two factor code. I don’t know what that is. I don’t have a two factor app anywhere on my devices. I can’t remember if I ever set that up years ago when I signed up. I lost most of my papers in my separation. Maybe there is a “recovery code” somewhere in them, or maybe not.

When I try “Try recovering your account”, all options on that screen are greyed out. “Cannot verify this device”, “SSH key”, “Personal access token”, all are grey. This is a kafkaesque nightmare. Why is this screen shown to me when there is literally no way to proceed? Who designed this?

I am still logged on GitHub on my iPad on Safari, but I don’t know how long this will last. I am afraid that next iOS update or something will make that go away. I will post this now while I still can.

Why isn’t just entering my username and password enough to log on GitHub any more?

**Comment by radar-blip** [comment]
If you are still logged in, you could try here: [Support page](https://support.github.com/contact/account)
Maybe support can at least point you in the right direction.

**Comment by airtower-luna** [comment]
If you're still logged in on another device you should be able to view your recovery codes or reconfigure/disable 2FA from there (go to Settings, and there to "Password and authentication"). Definitely **do not** log out, because a logged-in device is one of the recovery options. Otherwise contacting support as @linuxlawson said is your best option.
